starter-project-web-react/components/mintesnot/auth/LoginForm.tsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from 'react'
+import Link from 'next/link'
+import GitHubIcon from '@mui/icons-material/GitHub'
+import GoogleIcon from '@mui/icons-material/Google'
+import TwitterIcon from '@mui/icons-material/Twitter'
+import { signIn } from 'next-auth/react'
+import { useFormik } from 'formik'
+import {
+  Button,
+  Grid,
+  TextField,
+  Typography,
+  Box,
+  CircularProgress,
+} from '@mui/material'
+import * as yup from 'yup'
+
+interface FormValue {
+  email: string
+  password: string
+}
+
+const LoginForm = () => {
+  const [INITIAL_STATE_VALUE, setValue] = useState({
```

Comment:
initial state should n't have to be a state variable, you can use a simple object